### PR TITLE
Apply new autofixes

### PR DESCRIPTION
**Pull request by Akuli**
PR with empty commit, will hopefully be automagically turned into something more.

### CODE REVIEW

**Comment by rdbende** [comment]
`--exit-zero-even-if-changed`

**Comment by Moosems** [comment]
Nice name Akuli XD.

**Comment by Akuli** [comment]
> `--exit-zero-even-if-changed`

I already added a hacky `|| true` but this would be better.

File: porcupine/pluginmanager.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def get_docstring(module_name: str) -> str:
     spec = importlib.util.find_spec(module_name)
     if spec is not None and spec.origin is not None and spec.has_location:
         try:
-            with open(spec.origin, "r") as file:
+            with open(spec.origin) as file:
```
Comment (Akuli) [inline porcupine/pluginmanager.py:35]: Not sure if I like this change, but it's fine.

**Comment by rdbende** [comment]
Otherwise it works 😅

File: porcupine/utils.py [hunk 2]
```diff
@@ -249,7 +244,7 @@ def invert_color(color: str, *, black_or_white: bool = False) -> str:
 
     # tkinter uses 16-bit colors, convert them to 8-bit
     r, g, b = (value >> 8 for value in widget.winfo_rgb(color))
-    return "#%02x%02x%02x" % (0xFF - r, 0xFF - g, 0xFF - b)
+    return "#{:02x}{:02x}{:02x}".format(0xFF - r, 0xFF - g, 0xFF - b)
```
Comment (Akuli) [inline porcupine/utils.py:247]: ewww, but whatever

Comment (Moosems) [reply porcupine/utils.py:247]: An f-string would probably be better :(.

**Pull request merged**